src/esportsbot/cogs/TwitterIntegrationCog.py

```diff
@@ -20,15 +20,15 @@ async def addtwitter(self, ctx, twitter_handle=None, announce_channel=None):
         if twitter_handle is not None and announce_channel is not None:
             if (twitter_handle.replace('_', '')).isalnum():
                 twitter_in_db = db_gateway().get('twitter_info', params={
-                    'guild_id': ctx.author.guild.id, 'twitter_handle': twitter_handle})
+                    'guild_id': ctx.author.guild.id, 'twitter_handle': twitter_handle.lower()})
                 if not bool(twitter_in_db):
                     cleaned_channel_id = get_cleaned_id(announce_channel)
                     channel_mention = self.bot.get_channel(
                         cleaned_channel_id).mention
                     previous_tweet_id = self.get_tweets(
                         twitter_handle)[0]['id']
                     db_gateway().insert('twitter_info', params={
-                        'guild_id': ctx.author.guild.id, 'channel_id': cleaned_channel_id, 'twitter_handle': twitter_handle, 'previous_tweet_id': previous_tweet_id})
+                        'guild_id': ctx.author.guild.id, 'channel_id': cleaned_channel_id, 'twitter_handle': twitter_handle.lower(), 'previous_tweet_id': previous_tweet_id})
                     await ctx.channel.send(f"{twitter_handle} is valid and has been added, their Tweets will be placed in {channel_mention}")
                 else:
                     await ctx.channel.send(f"{twitter_handle} is already configured to output to {self.bot.get_channel(int(twitter_in_db['channel_id'])).mention}")
```

Comment:
```suggestion
                    await ctx.channel.send(f"{twitter_handle} is already configured to output to <#{twitter_in_db['channel_id']}>")
```
avoid `KeyError`s on cache misses by manually constructing a mention